recipes/libmeshb/all/test_package/test_package.c

```diff
@@ -0,0 +1,12 @@
+#include <libmeshb7.h>
+
+#include <stdio.h>
+
+int main() {
+   int64_t OutMsh;
+   if(!(OutMsh = GmfOpenMesh("test.meshb", GmfWrite, 2, 3)))
+      return 1;
+   GmfCloseMesh(OutMsh);
+   puts("Success! Created test.meshb");
```

Comment:
```suggestion
   GmfCloseMesh(NULL);
```

I would prefer not touching the disk if possible. The test package only needs to validate the library, do not need to be functional.

Reply:
Hi @uilianries doing that make the test_package failed with error code 139.
I don't know how do other IO library.
Perhaps we can skip this test_package validation?

Reply:
Thanks it works.